docs/docusaurus/docs/gx_cloud/tutorials/getting_started/cloud_quickstart.md

```diff
@@ -0,0 +1,305 @@
+)
+
+
+# Add Expectation
+expectation_configuration = gx.core.ExpectationConfiguration(**{
+  "expectation_type": "expect_column_values_to_not_be_null",
+  "kwargs": {
+    "column": "name",
+  },
+  "meta":{}
+})
+
+expectation_suite.add_expectation(
+    expectation_configuration=expectation_configuration
+)
+print(expectation_suite)
+context.save_expectation_suite(expectation_suite)
+
+
+# Validate data
+checkpoint_name = "my_quickstart_checkpoint"
+checkpoint_config = {
+  "name": checkpoint_name,
+  "validations": [{
+      "expectation_suite_name": expectation_suite.expectation_suite_name,
+      "expectation_suite_ge_cloud_id": expectation_suite.ge_cloud_id,
+      "batch_request": {
+          "datasource_name": datasource.name,
+          "data_connector_name": "default_runtime_data_connector",
+          "data_asset_name": "my_quickstart_data_asset",
+      },
+  }],
+  "config_version": 1,
+  "class_name": "Checkpoint"
+}
+
+context.add_or_update_checkpoint(**checkpoint_config)
+checkpoint = context.get_checkpoint(checkpoint_name)
+print(checkpoint)
+
+
+# View results
+import pandas as pd
+
+df = pd.DataFrame({'name': ["apple", "banana", "cherry"], 'quant': [40, 50, 60]}) 
+batch_request = {
+    "runtime_parameters": {
+        "batch_data": df
+    },
+    "batch_identifiers": {
+        "my_quickstart_identifier": "test"
+    }
+}
+
+result = context.run_checkpoint(ge_cloud_id=checkpoint.ge_cloud_id, batch_request=batch_request)
+print(result)
+```
+
+In the following steps we'll break down exactly what is happening here so that you can follow along and perform a <TechnicalTag tag="validation" text="Validation"/> yourself.
+
+## Steps
+
+### 1. Setup
+
+#### 1.1 Generate User Token
+
+Go to [“Settings” > “Tokens”](https://app.greatexpectations.io/tokens) in the navigation panel and generate User Token. 
+These tokens are see-once and stored as a hash in Great Expectation Cloud's backend database. Once a user copies their API key, the Cloud UI will never show the token value again. 
+
+#### 1.2 Create Data Context
+
+:::tip 
+Any Python Interpreter or script file will work for the remaining steps in the guide, we recommend using a Jupyter Notebook, since they are included in the OSS GX installation and give the best experience of both composing a script file and running code in a live interpreter.
+:::
+
+Paste this snippet into the next notebook cell to instantiate Cloud <TechnicalTag tag="data_context" text="Data Context"/>:
+
+```python title="Jupyter Notebook"
+import great_expectations as gx
+
+ context = gx.get_context(
+    cloud_access_token = "<user_token_you_generated_in_the_app>",
+    cloud_organization_id = "<organization_id_from_the_app>",
+)
+```
+
+### 2. Connect to data
+
+Modify the following yaml code to connect to your datasource. Otherwise, for the purpose of this guide, you can use this simple Pandas <TechnicalTag tag="datasource" text="Datasource"/> configuration to connect to:
```

Comment:
```suggestion
Modify the following yaml code to connect to your <TechnicalTag tag="datasource" text="Datasource"/> . Otherwise, for the purpose of this guide, you can use the following Pandas Datasource configuration to connect to an in-memory Pandas DataFrame:
```